meilisearch-lib/src/index_controller/dump_actor/loaders/v1.rs

```diff
@@ -34,208 +20,5 @@ impl MetadataV1 {
-            ranking_rules: match settings.ranking_rules {
-                Some(Some(ranking_rules)) => Setting::Set(ranking_rules.into_iter().filter_map(|criterion| {
-                    match criterion.as_str() {
-                        "words" | "typo" | "proximity" | "attribute" | "exactness" => Some(criterion),
-                        s if s.starts_with("asc") => compat::asc_ranking_rule(s).map(|f| format!("{}:asc", f)),
-                        s if s.starts_with("desc") => compat::desc_ranking_rule(s).map(|f| format!("{}:desc", f)),
-                        "wordsPosition" => {
-                            warn!("The criteria `attribute` and `wordsPosition` have been merged \
-                                into a single criterion `attribute` so `wordsPositon` will be \
-                                ignored");
-                            None
-                        }
-                        s => {
-                            error!("Unknown criterion found in the dump: `{}`, it will be ignored", s);
-                            None
-                        }
-                    }
-                }).collect()),
-                Some(None) => Setting::Reset,
-                None => Setting::NotSet
-            },
-            // we need to convert the old `Vec<String>` into a `BTreeSet<String>`
-            stop_words: match settings.stop_words {
-                Some(Some(stop_words)) => Setting::Set(stop_words.into_iter().collect()),
-                Some(None) => Setting::Reset,
-                None => Setting::NotSet
-            },
-            // we need to convert the old `Vec<String>` into a `BTreeMap<String>`
-            synonyms: match settings.synonyms {
-                Some(Some(synonyms)) => Setting::Set(synonyms.into_iter().collect()),
-                Some(None) => Setting::Reset,
-                None => Setting::NotSet
-            },
-            _kind: PhantomData,
-        }
-    }
-}
-
-/// Extract Settings from `settings.json` file present at provided `dir_path`
-fn import_settings(dir_path: impl AsRef<Path>) -> anyhow::Result<Settings> {
-    let path = dir_path.as_ref().join("settings.json");
-    let file = File::open(path)?;
-    let reader = std::io::BufReader::new(file);
-    let metadata = serde_json::from_reader(reader)?;
-
-    Ok(metadata)
-}
-
-#[cfg(test)]
-mod test {
-    use super::*;
-
-    #[test]
-    fn settings_format_regression() {
-        let settings = Settings::default();
-        assert_eq!(
-            r##"{"rankingRules":null,"distinctAttribute":null,"searchableAttributes":null,"displayedAttributes":null,"stopWords":null,"synonyms":null,"attributesForFaceting":null}"##,
-            serde_json::to_string(&settings).unwrap()
-        );
-    }
+        anyhow::bail!("The version 1 of the dumps is not supported anymore. You can re-export your dump from a version between 0.21 and 0.24, or start fresh from a version 0.25 onwards.")
```

Comment:
The `0.25` will stay even if the user tries to pass to v0.26.0 or later. Can we dynamically change this part, or re-do the sentence?

Reply:
the support has been dropped in 0.25, hence "0.25 onward"

Reply:
what do you want to convey @curquiza ?

Reply:
oh yes sorry, I completely missed the onwards! my bad!